components/public-api-server/pkg/oidc/router.go

```diff
@@ -32,8 +32,9 @@ func Router(s *Service) *chi.Mux {
 }
 
 const (
-	stateCookieName = "state"
-	nonceCookieName = "nonce"
+	stateCookieName    = "state"
```

Comment:
nit: It bet these names work because they are basically very short-lived. And changing them now does not make a ton of sense, because we just repeat the pattern. In general I would love to prefix them somehow.